src/backend_connection/authenticated_cmds.rs

```diff
@@ -24,7 +24,8 @@ use crate::{
 };
 
 #[pyclass]
-pub(crate) struct AuthenticatedCmds(Arc<client_connection::AuthenticatedCmds>);
+#[derive(Clone)]
+pub(crate) struct AuthenticatedCmds(pub Arc<client_connection::AuthenticatedCmds>);
```

Comment:
If you put `AuthenticatedCmds` into an Arc, why make it derive `Clone` ?

Reply:
Nice catch, forgot to remove